<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoToDialog</class>
 <widget class="QDialog" name="GoToDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Go To</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="orderLabel">
       <property name="text">
        <string>Order</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="orderSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="stepLabel">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="stepSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="trackLabel">
       <property name="text">
        <string>Track</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="trackComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GoToDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GoToDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
